src/middlewared/middlewared/plugins/disk_/disk_info.py

```diff
@@ -1,17 +1,14 @@
 import os
 import re
-from xml.etree import ElementTree as etree
 
 import bsd
-import sysctl
 from middlewared.service import Service
-from .disk_info_base import DiskInfoBase
```

Comment:
We are making changes to not consume `DiskInfoBase`, why keep those files still ( `disk_info_base.py` ) ? Although probably better/nicer to make these cleanup changes in a different PR :)